Formula/s-nail.rb

```diff
@@ -0,0 +1,28 @@
+class SNail < Formula
+  desc "Fork of Heirloom mailx."
+  homepage "https://www.sdaoden.eu/code.html"
+  url "https://www.sdaoden.eu/downloads/s-nail-14.9.3.tar.gz"
+  sha256 "9048abe94c8b732ddefcd70b9f6052da16977de76b07598790e4c763e97f911d"
+
+  depends_on "openssl"
+  depends_on "libidn"
+
+  def install
+    mv "INSTALL", "INSTALL.txt"
+    system "make", "OPT_AUTOCC=no",
+                   "CC=#{ENV.cc}",
+                   "cc_maxtopt=1",
+                   "OPT_NOMEMBDBG=1",
+                   "C_INCLUDE_PATH=#{Formula["openssl"].opt_include}",
+                   "LDFLAGS=-L#{Formula["openssl"].opt_lib}",
+                   "VAL_PREFIX=#{prefix}",
+                   "OPT_DOTLOCK=no",
+                   "config"
+    system "make", "build"
+    system "make", "install"
+  end
+
+  test do
+    system "#{bin}/s-nail", "-Xversion", "-Xx"
```

Comment:
The test should exercise and verify the functionality of the software once installed.